experimental/traffic-portal/src/app/shared/import-json-txt/import-json-txt.component.html

```diff
@@ -0,0 +1,38 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<h2 mat-dialog-title>{{data.title}}</h2>
+<mat-dialog-content id="import-content" [ngClass]="{'active':dragOn}">
+	<input type='file' id="imageUpload" aria-label="Select a file" accept="application/json,text/plain" #fileInput (change)="uploadFile($event)"/>
+	<button type="button" class="dropzone" (click)="fileInput.click()">
+		<div class="text-wrapper">
+			<div class="label">
```

Comment:
[a button may only contain phrasing content](https://html.spec.whatwg.org/#the-button-element). I think you probably want to make this instead a `label` of the `input` and put both within one or more flow content elements (e.g. `div`) that listen for drag events.

Drag events should set the value of the input, by the way, since otherwise it could appear to users that dragging a file into the space doesn't change the input, which is confusing because then which is uploaded?